include/shared.h

```diff
@@ -89,6 +89,8 @@ void ft_parse_addr_opts(int op, char *optarg, struct cs_opts *opts);
 void ft_parsecsopts(int op, char *optarg, struct cs_opts *opts);
 void ft_usage(char *name, char *desc);
 void ft_csusage(char *name, char *desc);
+void ft_integ_fill_buf(void *buf, int size, int iter);
+int ft_integ_check_buf(void *buf, int size, int iter);
```

Comment:
`integ` was for "data integrity", Prankur can drop the name though, it's not a big deal